docs/cache.txt

```diff
@@ -0,0 +1,240 @@
+.. _laravel-queues:
+
+===============
+Cache and Locks
+===============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: php framework, cache, lock, code example
+
+Configuration
+-------------
+
+To use MongoDB as a backend for `Laravel Cache and Locks <https://laravel.com/docs/{+laravel-docs-version+}/cache>`__,
+add a store configuration by using the ``mongodb`` driver in ``config/cache.php``:
+
+.. code-block:: php
+
+   'stores' => [
+       'mongodb' => [
+           'driver' => 'mongodb',
+           'connection' => 'mongodb',
+           'collection' => 'cache',
+           'lock_connection' => 'mongodb',
+           'lock_collection' => 'cache_locks',
+           'lock_lottery' => [2, 100],
+           'lock_timeout' => 86400,
+       ],
+   ],
+
+To configure the ``mongodb`` database connection, see the :ref:`laravel-fundamentals-connection` section.
+
+The following table describes a list of cache and lock options
+and their default values:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * - Setting
+     - Description
+
+   * - ``driver``
+     - **Required**. Specifies the lock driver to use. Must be ``mongodb``.
+
+   * - ``connection``
+     - **Required**. The database connection used to store cache items. It must be a ``mongodb`` connection.
+
+   * - ``collection``
+     - Default ``cache``. Name of the MongoDB collection to store cache items.
```

Comment:
I didn't pick up on this when reviewing the original PRs, so forgive me for asking this late: where does the database name come from? Are you relying on whatever is configured through the corresponding `connection` option (for both this and `lock_collection`)?

Reply:
You can also consider allowing a namespace for the `collection` and `lock_collection` options, and parsing out a database name by exploding on a dot character (no issue since it's prohibited in each part per [Naming Restrictions](https://www.mongodb.com/docs/manual/reference/limits/#naming-restrictions).

Otherwise, I'm fine keeping this as-is and revisiting if/when someone asks for further customization.

Reply:
The default is applied in the service provider.
https://github.com/mongodb/laravel-mongodb/blob/f95eb66f5920a368cf665fe3fb4d158558745a67/src/MongoDBServiceProvider.php#L51

Reply:
Tracking this feature: PHPORM-176